src/Microsoft.FeatureManagement/Microsoft.FeatureManagement.csproj

```diff
@@ -18,19 +18,22 @@
     <PackageIconUrl>https://aka.ms/AzureAppConfigurationPackageIcon</PackageIconUrl>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
   </PropertyGroup>
-  
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
+  </ItemGroup>
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.1.10" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.5" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.5" />
```

Comment:
Bumping the major version of dependencies can lead to breaking changes. We would need to bump the major version of our package as well. I'm also not sure how requiring 3.x.x packages versions is going to behave for those who are running ASP.NET Core 2.1. As far as I know, the 2.1.x packages are still supported.

Reply:
Let's just skip the major version bump for these. 

2.1.x is still supported, and those who need any new APIs exposed in 3.1.x packages will most likely already have references to them resulting in auto upgrade.

Reply:
It should not affect this PR so it makes sense to don't update them here. And you are right that it could be overridden by users. I've reverted to the version that was used before. Thank you

Reply:
Also about NuGet package support, if I'm not mistaken, Logger and other Extension packages support are tied to .net core support, and since net core 3.0 already out of support those packed also not supported so I would suggest updating them (in a separate PR).

Reply:
.NET Core 2.1 is still in support so Microsoft.Extensions... 2.1.x packages are still supported.

Reply:
Ah, sorry I've skipped the dates for net core 2.1